src/Middleware/HttpLogging/src/W3CLogger.cs

```diff
@@ -35,10 +35,10 @@ internal virtual W3CLoggerProcessor InitializeMessageQueue(IOptionsMonitor<W3CLo
 
     public void Log(string[] elements)
     {
-        _messageQueue.EnqueueMessage(Format(elements));
+        _messageQueue.EnqueueMessage(Format(elements.AsSpan()));
```

Comment:
Why change to a Span?

Reply:
I did not notice this commit on this PR 😅 . I was designed how we can use Span, on the format message (while writing directly to the `StreamWriter`. But in a second PR for the same issue 🙂 .